src/tests/Microsoft.PowerFx.Core.Tests/ExpressionTestCases/RoundUp.txt

```diff
@@ -141,7 +141,7 @@
 // Decimal NUMBER and NUMBER_OF_DIGITS parameter
 
 >> RoundUp(14.123456,1111111111)
-14.123456
+Error({Kind:ErrorKind.Numeric})
```

Comment:
Do we need to make this an error? Decimal numbers have up to 28 significant digits (and floating point up to ~15), so this is equivalent to RoundDown(14.123456, 15) or RoundDown(14.123456, 28) for float and decimal respectively, which we can compute.